<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Central Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>510</height>
     </rect>
    </property>
    <widget class="QLabel" name="TitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Central Application</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>139</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Items</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>139</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Button 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>139</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Button 3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>169</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Button 4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>169</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Button 5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>169</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Button 6</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ItemWindow" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>551</width>
      <height>511</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="PLU_Input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Description_Input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Send_Item">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>39</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
